src/test/java/survey/TestWebApp.java

```diff
@@ -1,6 +1,5 @@
 package survey;
```

Comment:
Commenting everything out of this file is okay if swagger will test this. Do we need to write the tests for swagger, and if so where?
If we can then we can probably just delete this file

Reply:
We do not need test for swagger, it is just a documentation tool.
Uncommented the test file.